demo/llama_stack_mcp_agent.py

```
@@ -0,0 +1,115 @@
+registered_tools = client.tools.list()
+registered_toolgroups = [t.toolgroup_id for t in registered_tools]
+
+if "mcp::openshift" not in registered_toolgroups:
+    # Register MCP tools
+    try:
+        client.toolgroups.register(
+            toolgroup_id="mcp::openshift",
+            provider_id="model-context-protocol",
+            mcp_endpoint=McpEndpoint(uri=mcp_url),
+        )
+        logger.info(f"MCP tools registered successfully")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)